pkg/jsonpretty/format.go

```diff
@@ -0,0 +1,143 @@
+package jsonpretty
```

Comment:
Do we want this package to be exported? If so, can we add a comment to the top here describing the purpose and functionality of the package?